repository/agenda-general.js

```diff
@@ -123,7 +123,14 @@ const selectAgendaitemsForSorting = async (agendaURI) => {
       ${sparqlEscapeUri(agendaURI)} dct:hasPart ?agendaitem .
       ?agendaitem a besluit:Agendapunt ;
         schema:position ?number ;
-        ext:wordtGetoondAlsMededeling ?isRemark .
+        dct:type ?type .
+      BIND(
+        IF (
+          ?type = <http://themis.vlaanderen.be/id/concept/agendapunt-type/8f8adcf0-58ef-4edc-9e36-0c9095fd76b0>,
+          "true"^^<http://mu.semte.ch/vocabularies/typed-literals/boolean>,
+          "false"^^<http://mu.semte.ch/vocabularies/typed-literals/boolean>
+        ) AS ?isRemark
```

Comment:
De omweg langs booleans in de query is niet meer nodig. Je kan in de code misschien beter vergelijken op de URI value dan op de boolean-as-string value zoals nu het geval is?